### ISSUE TITLE
Session 05 SEE & SAY Additions: 1) Episodes of Care, 2) Community Button, 3) Tutor, 4) Bug Reporting, 5) Learning Mode

### ISSUE THREAD

**Issue by staceypark** [lzim/teampsd#1043]
**1. Add description:** 
We need updates in the MTL 2.0 SEE and SAY guides for session 5 describing:
1) Episodes of Care
From Debbie in #648 
> Hi TomRust - An action item in Lucid says, "Also need to comb through an look for any mentions that states MTL units are "Patients" or "Appointments" because we also have the unit: "Episodes of Care" for SP." @branscombj and I went through the SP module and found 2 occurrences of Episodes of Care on the model diagram linking dials or rectangle measured in patients; 2 mentions of Episodes of Care in the team data table measured in patients/week or weeks, and one 1 experimental variable measured in patients/week. Screen shots below. Not sure how to best edit the see/say files given the Episodes of Care are measured mainly in patients/week or weeks. Talked with @jamesmrollins @lzim about this today and suggestion was to put the information here. Thanks.
![image](https://user-images.githubusercontent.com/31182258/73580549-f0bb4300-4453-11ea-84e9-e1ea5c1b69c9.png)
![image](https://user-images.githubusercontent.com/31182258/73580567-fdd83200-4453-11ea-8975-6bb0eb9c1a71.png)

2) Community Button 
3) Tutor Button (Note: Tutor function is only available in tablet or desktop, but not on smart phone.)
4) Bug reporting should be added to in detail based on testing of how copy/paste works.
5) Learning Mode
 
**2. Click on the right:**  
i. **Pipelines** - prioritize in a TeamPSD (& Workgroup) pipeline  
ii. **Assignee** - assign to an individual team member producing work for the issue  
iii. **Label**- assign workgroup (lead) responsible for tracking the task  
iv. **Projects** - assign to domain-specific TeamPSD tracker (issues, features, document, manuscript, team)  
v. **Milestones** - select monthly sprint milestone within the workgroup   
vi. **Estimate** - Estimate task in complexity points   
vii. **Epics** - assign to team-wide epic and TeamPSD master plan project  
viii. **Releases** - assign to key team release grouped with other relevant issues  

**3. Once you hit "Submit new issue**, underneath the first comment:   
viiii. **Add dependency** - assess for and add dependencies *blocking* & *blocked by* this issue

**Comment by staceypark** [comment]
Included News flashes, andon alert, changes to chat -> team, and help -> more info.
